lib/Extension/Bundle/Installer/SettingsStoreAwareInstaller.php

```diff
@@ -142,18 +142,27 @@ public function uninstall()
         $this->markUninstalled();
     }
 
+    /**
+     * @return bool
+     */
     public function isInstalled()
     {
         $installSetting = SettingsStore::get($this->getSettingsStoreInstallationId(), 'pimcore');
 
-        return $installSetting ? $installSetting->getData() : false;
+        return (bool) ($installSetting ? $installSetting->getData() : false);
     }
 
+    /**
+     * @return bool
+     */
     public function canBeInstalled()
     {
         return !$this->isInstalled();
     }
 
+    /**
+     * @return bool|float|int|string
```

Comment:
sorry, missed that one before ... here we also should just allow bool as return value ...

Reply:
Changed this one as well.